File: 66-plus-one/plus-one.js
/**
 * @param {number[]} digits
 * @return {number[]}
 */
var plusOne = function(arr) {

    let res = 0;
    let str = '';

    // res = Number(arr.join(''));   //-- if not using the below for loop...then uncomment..
    for(let i=0;i<arr.length;i++) {
        str+=arr[i];
    }
    res = BigInt(str); //-- if not using for loop ,then comment this ...
    res++;

   return res.toString().split('').map(Number);


    
};

File: 66-plus-one/plus-one.java

import java.math.BigInteger;
class Solution {
    public int[] plusOne(int[] digits) {

        String x = "";

        for(int i=0;i<digits.length;i++) x+=digits[i];
        

        BigInteger res = new BigInteger(x);
        res = res.add(BigInteger.ONE);

        System.out.println(res);

        return res.toString().chars().map(Character :: getNumericValue).toArray();


        
    }
}